probe/tag/pidtree.go

```diff
@@ -24,7 +24,7 @@ var (
 	readFile = ioutil.ReadFile
 )
 
-func newPIDTree(procRoot string) (*pidTree, error) {
+func newRealPIDTree(procRoot string) (*pidTree, error) {
```

Comment:
- var newFoo = {newMockFoo, newRealFoo}
- var newFooStub = {newMockFoo, newFoo}

The first option is just more symmetric. Similarly I don't like invoking newFooStub in my actual code :)